pkg/ipamclaimswebhook/podmutator_test.go

```diff
@@ -111,25 +111,80 @@ var _ = Describe("KubeVirt IPAM launcher pod mutato machine", Serial, func() {
 			Equal(config.expectedAdmissionResponse),
 		)
 	},
-		Entry("pod not requesting secondary attachments is accepted", testConfig{
+		Entry("pod not requesting secondary attachments and no primary user defined network is accepted", testConfig{
 			inputVM:  dummyVM(nadName),
 			inputVMI: dummyVMI(nadName),
-			inputNAD: dummyNAD(nadName),
+			inputNADs: []*nadv1.NetworkAttachmentDefinition{
+				dummyNAD(nadName),
+			},
 			inputPod: &corev1.Pod{},
 			expectedAdmissionResponse: admission.Response{
 				AdmissionResponse: admissionv1.AdmissionResponse{
 					Allowed: true,
 					Result: &metav1.Status{
-						Message: "no secondary networks requested",
+						Message: "not a VM",
```

Comment:
what ? I just don't understand how a test about "pod not requesting secondary attachments and no primary user defined network is accepted" can expect to have a "not a VM" result.

The description should match the expectation - i.e. "non VM pod ..."

Reply:
Adding to the description `pod not beloging to a VM` like other tests.